bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -243,6 +252,15 @@ pub mod pallet {
 	pub type PricingParameters<T: Config> =
 		StorageValue<_, PricingParametersOf<T>, ValueQuery, T::DefaultPricingParameters>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn tokens)]
+	pub type Tokens<T: Config> = StorageMap<_, Twox64Concat, TokenId, Location, OptionQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn location_tokens)]
+	pub type LocationToToken<T: Config> =
+		StorageMap<_, Twox64Concat, Location, TokenId, OptionQuery>;
```

Comment:
Same here: Implicit use of XCM v4 as a storage key.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/5546/commits/5ef32c8729f8f1f8c41b82a2d87f99e65f450c2c